ArduCopter/GCS_Mavlink.cpp

```diff
@@ -610,22 +610,29 @@ MAV_RESULT GCS_MAVLINK_Copter::handle_command_int_packet(const mavlink_command_i
 
 MAV_RESULT GCS_MAVLINK_Copter::handle_command_mount(const mavlink_command_long_t &packet)
 {
-    // if the mount doesn't do pan control then yaw the entire vehicle instead:
+    AP_Mount *mount = AP::mount();
+    // if the mount doesn't do pan control or mount is in follow-the-vehicle mode, then yaw the entire vehicle instead:
     switch (packet.command) {
-#if MOUNT == ENABLED
```

Comment:
Please leave the macro checks in place

Reply:
Macros are back in

Reply:
This should be fixed as of 0dbd556cc50e388dbbef4c797a57b802456a0631

Reply:
@anthonygillespie I don't believe it is.  If you look at the diff to the original code you'll see the `MOUNT` macro check is outside of the `MAV_CMD_DO_MOUNT_CONTROL` case statement check.  Basically: the diff from this PR should not modify the location of the `MOUNT` macro check.

Reply:
I can move the macros back to where they were, but I guess I don't understand the purpose of these macros.  When would a user not define MOUNT?  Which code would not compile if MOUNT isn't defined?  It seems like mount and copter.camera_mount might not compile if MOUNT isn't set, but are you saying MAV_CMD_DO_MOUNT_CONTROL might not be defined either?

Reply:
Correct. And, we want to minimize the delta between upstream.

Reply:
Ok, just moved the macros to include the case statements